source/user-manual/ruleset/ruleset-xml-syntax/decoders.rst

```diff
@@ -8,30 +8,80 @@ Decoders Syntax
++-------------------+---------------------------------------------------------------+-------------------------------------------------------------------------------------------------+
+| `ftscomment`_     | Any String                                                    | Adds a comment to fts.                                                                          |
++-------------------+---------------------------------------------------------------+-------------------------------------------------------------------------------------------------+
+| `plugin_decoder`_ | See `below <decoders.html#plugin-decoder>`_                   | Specifies a plugin that will do the decoding. Useful when extraction with regex is not          |
+|                   |                                                               | feasible.                                                                                       |
++-------------------+---------------------------------------------------------------+-------------------------------------------------------------------------------------------------+
+| `use_own_name`_   | True                                                          | Only for child decoders.                                                                        |
++-------------------+---------------------------------------------------------------+-------------------------------------------------------------------------------------------------+
+| `json_null_field`_| String                                                        | Adds the option of deciding how a null value from a JSON will be stored.                        |
++-------------------+---------------------------------------------------------------+-------------------------------------------------------------------------------------------------+
+| `var`_            | Name for the variable.                                        | Defines variables that can be reused inside the same file.                                      |
++-------------------+---------------------------------------------------------------+-------------------------------------------------------------------------------------------------+
+
+How it works
+------------
+
+To understand the inner workings of a decoder, it will be easier through examples like the following:
+
+::
+
+  Apr 14 19:28:21 gorilla sshd[31274]: Connection closed by 192.168.1.33
+
+  **Phase 1: Completed pre-decoding.
+         full event: 'Apr 14 19:28:21 gorilla sshd[31274]: Connection closed by 192.168.1.33'
+         timestamp: 'Apr 14 19:28:21'
+         hostname: 'gorilla'
+         program_name: 'sshd'
+         log: 'Connection closed by 192.168.1.33'
+
+  **Phase 2: Completed decoding.
+         decoder: 'sshd'
+         srcip: '192.168.1.33'

```

Comment:
I think this should be a code-block with class `output`

Reply:
You are right. I've just fixed it.